.changeset/smooth-turkeys-grab.md

```diff
@@ -0,0 +1,6 @@
+---
+"hardhat": patch
+"docs": patch
```

Comment:
I hate that the link adds docs autmatically. It breaks changesets.
```suggestion
```

Reply:
Oops, didn't notice that.